JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
As written, these are actually describing own data properties of the `Module` instance objects.  One can use internal hooks for data properties to do lazy allocation on the first read, so that isn't an issue, but maybe it is a bit simpler and more explicit to express this lazy allocation as a getter...